docker/common.sh

```diff
@@ -5,7 +5,7 @@ function run_deployment_tests() {
   export XRT_WORKERS="localservice:0;grpc://localhost:40934"
   export CC=clang-8 CXX=clang++-8
 
-  time python /pytorch/xla/test/test_train_mnist.py
+  time python /pytorch/xla/test/test_train_mp_mnist.py --fake_data
```

Comment:
if the previous command has early stopping (using convergence), and this doesn't, it may have different behavior.

Reply:
No early stopping pre-existing. This is mainly a very simple test to block deployment of wheels if we smell any smoke.